library/src/blas1/rocblas_rot_batched.cpp

```diff
@@ -57,7 +57,7 @@ namespace
                       incx,
                       "--incy",
                       incy,
-                      "--batch",
+                      "--batch_count",
```

Comment:
NOT for this PR but should have created constants for all these standardized parameter strings.

Reply:
> NOT for this PR but should have created constants for all these standardized parameter strings.

That would require a large refactoring, and may reduce readability if variables are used everywhere that string literals are used now. It may also require compile-time or run-time concatenation of names and option dashes "--". It would also have to span multiple languages (C++, Python, YAML). Right now the strings seem to be sufficient.